src/components/PrevNextTiles.tsx

```diff
@@ -5,6 +5,7 @@ import type { SidebarConfig, SidebarGroup, SidebarLink } from '@/types'
 type Props = {
   config?: SidebarConfig
   currentPath: string
+  hideSecondarySidebar?: boolean
```

Comment:
Sounds weird because those tiles don't have anything to do with the secondary sidebar. I'd call it something like `isFullWidth`


```suggestion
  isFullWidth?: boolean
```